GOVERNANCE.md

```
@@ -1,100 +1,154 @@
 
----
+We value contributions from humans who understand what they're submitting, even if they had some algorithmic assistance along the way.
+The goal is quality and genuine engagement with the project, not quantity of AI-generated content.
 
-## Lazy Consensus
+### Using LLMs for Reviews
 
-ESO uses [Lazy Consensus](http://en.osswiki.info/concepts/lazy_consensus) for most decisions.
+We welcome the use of AI to review work, as it might help us follow our policy here above.
+
+We are currently evaluating CodeRabbit https://app.coderabbit.ai/login and GitHub Copilot for our contributions.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)